migrations/src/main/resources/application.properties.template

```diff
@@ -13,8 +13,8 @@
 # limitations under the License.
 #
 
-menas.mongo.connection.string=mongodb://localhost:27017
-menas.mongo.connection.database=menas
+enceladus.rest.mongo.connection.string=mongodb://localhost:27017
```

Comment:
why not `enceladus.rest_api.` as on other places? I'm just trying to understand & follow the consistent conventions. Or, the other way around, why not to have `enceladus.rest.` everywhere, instead of these `enceladus.rest_api.`?

Reply:
(just a note - also on other places, if we decide to change this)

Reply:
Not sure how it would then work in Docker's env-variables setup of these (as there the separator is `_` instead of `.`).